src/styles/overrides/_se-events.scss

```diff
@@ -0,0 +1,60 @@
+.dg_actions-list {
+    ul {
+        list-style-type: none;
+        margin: 0;
+        padding: 0;
+
+        li {
+            display: inline-block;
+            margin-right: $default-margin;
+            margin-bottom: $default-margin;
+
+            a {
+                @extend .dg_button;
+                display: inline-block;
+
+                &:before {
+                    margin-right: $margin-smallest;
+                }
+            }
+
+            &.action a {
+                background-image: none;
```

Comment:
What is the function of `background-image: none;`? Hid it with inspect element, and didn't notice any adverse interactions.